### DISCUSSION TITLE
Need help setting DecapCMS with private github Repos

### DISCUSSION THREAD

**Discussion by reddypramod85** [decaporg/decap-cms#7340] [Q&A]
Hi Team,

Trying to use DecapCMS with private github repo and running into below issue.

<img width="949" alt="image" src="https://github.com/user-attachments/assets/8f7ecd9f-50d8-44b9-abb5-b6da1558fc28">

Here is the config.yaml file for reference.

backend:
 name: github
 repo: hpe-dev-incubator/hpe-dev-portal-public
 branch: master
 auth_scope: repo
 open_authoring: true

publish_mode: editorial_workflow

**Comment by alfredlinson** [comment]
I have seen this error. Mine was also a private repo inside the organization. I was using an external OAuth setup, so the domain name of base_url in config.yaml and the callback URL in the OAuth app should be the same. Initially, I put it wrong and got the same error screen as you have attached. If you are using external OAuth, then make sure the URLs match.
